src/main/docs/guide/annotationConfiguration/openapimanagement.adoc

```diff
@@ -0,0 +1,19 @@
+api:openapi.annotation.OpenAPIManagement[] adds management endpoints.
+
+api:openapi.annotation.OpenAPIManagement[] is mapped to: 
+
+```java
```

Comment:
It'd be better to use asciidoctor format, not markdown

Reply:
Changed